lm_eval/models/ibm_watsonx_ai.py

```diff
@@ -0,0 +1,402 @@
+import json
+import os
+from configparser import ConfigParser
+from functools import cache
+from pathlib import Path
+from typing import Any, Dict, NamedTuple, Optional, Type, cast
+
+from tqdm import tqdm
+
+
+try:
```

Comment:
I think you need to move this global scope to inner scope, as the tests do not install extension by defaults.

https://github.com/EleutherAI/lm-evaluation-harness/blob/c1d8795da7610d507cb191c2769c5e7bf1060a35/lm_eval/models/anthropic_llms.py#L46

Reply:
Done, moved imports to inner scope